base/array.jl

```diff
@@ -713,15 +748,14 @@ Resize `a` to contain `n` elements. If `n` is smaller than the current collectio
 length, the first `n` elements will be retained. If `n` is larger, the new elements are not
 guaranteed to be initialized.
 
+# Examples
 ```jldoctest
 julia> resize!([6, 5, 4, 3, 2, 1], 3)
 3-element Array{Int64,1}:
  6
  5
  4
-```
 
-```julia-repl
 julia> resize!([6, 5, 4, 3, 2, 1], 8)
```

Comment:
this isn't guaranteed to always be zero-initialized beyond the end of the array, is it?